FAQs.md

```diff
@@ -0,0 +1,21 @@
+# FAQ
+
+## AtomOne vs Gaia
+
+AtomOne is designed as an alternative fork of Gaia with a distinct focus on
+minimalism in its ICS & Token IBC hub. This results in a leaner, more efficient
+architecture. In terms of governance, AtomOne introduces a higher consensus
+threshold (Constitutional Majority) and emphasizes decentralized
+decision-making. In terms of software, AtomOne will start as a fork of
+"cosmoshub4".
```

Comment:
```suggestion
"cosmoshub-4".
```

Reply:
This is also done in other places. But the correct chain ID has a dash
https://github.com/cosmos/chain-registry/blob/master/cosmoshub/chain.json